src/Modules/Data/Commands.Data/help/Add-PowerBIRows.md

```diff
@@ -0,0 +1,152 @@
+---
+external help file: Microsoft.PowerBI.Commands.Data.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Data
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIRows
+
+## SYNOPSIS
+Insert rows to the specified table in Power BI.
+
+## SYNTAX
+
+### Dataset (Default)
+```
+Add-PowerBIRows -Dataset <Dataset> -TableName <String> -Rows <JArray> [-WorkspaceId <Guid>]
+ [-Workspace <Workspace>] [<CommonParameters>]
+```
+
+### DatasetId
+```
+Add-PowerBIRows -DatasetId <Guid> -TableName <String> -Rows <JArray> [-WorkspaceId <Guid>]
+ [-Workspace <Workspace>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Add-PowerBIRows creates new rows to the specified table.
```

Comment:
Creates new rows in the specified Power BI table.

Reply:
Done